src/test/java/winstone/HttpConnectorFactoryTest.java

```diff
@@ -49,6 +57,20 @@ public void testListenAddress() throws Exception {
 
     @Test
     public void writePortInFile() throws Exception {
+        Callable<String> parallelVerifier = () -> {
+            System.out.println("Async Verifier started!");
+            Path portFile = Paths.get(tmp.getRoot().getAbsolutePath(), "subdir/jenkins.port");
+
+            Awaitility.await()
+                    .pollInterval(1, TimeUnit.MICROSECONDS)
+                    .atMost(5, TimeUnit.SECONDS)
+                    .until(() -> Files.exists(portFile));
+
+            return FileUtils.readFileToString(portFile.toFile(), StandardCharsets.UTF_8);
```

Comment:
This repository is using Java 11, so no need for a third-party library anymore!

```suggestion
            return Files.readString(portFile, StandardCharsets.UTF_8);
```